app/src/main/java/com/amaze/filemanager/filesystem/HybridFile.java

```diff
@@ -620,6 +621,50 @@ public Long execute(SFTPClient client) throws IOException {
     }
 
 
+    public long folderSize(Context context,  final OnProgressUpdate<Long[]> updateState, Long[] spaces) {    
```

Comment:
If you can remove the `Long[] spaces` from the signature, `folderSize(Context)` can become:

```java
public long folderSize(Context context) {
    return folderSize(context, null);
}

```